listVideo.js

```diff
@@ -0,0 +1,45 @@
+import Sequelize from 'sequelize';
+import Video from './db/models/video';
+import db from './config/db';
+
+export async function main(event, context, callback) {
+  // Do not wait for sequelize connection to close
+  context.callbackWaitsForEmptyEventLoop = false;
+
+  const headers = {
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Credentials': true,
+  };
+
+  Video.init(db);
+
+  let response;
+  let statusCode;
+
+  try {
+    const result = await Video.findAll();
+    statusCode = 200;
+    let body = JSON.stringify(result);
+
+    if (result.length === 0) {
+      statusCode = 204;
```

Comment:
I think you can return 200 and just return an empty list here.

Reply:
That's also possible. Simple.